### ISSUE TITLE
Unable to keep up the language setting as English mode among the applications

### ISSUE THREAD

**Issue by czchen** [chewing/ibus-chewing#41]
From https://bugs.launchpad.net/ubuntu/+source/ibus-chewing/+bug/1293506

I can reprocedure this scenario in debian with the following version:
ibus-chewing 1.4.10.1-1
ibus 1.5.5-1

[Test Case]
- On the language preferencese setting, will set to "Share language setting among applications" option true.
- Lanuch the gedit text editor.
- On the top menu bar, click the language tray icon.
- Enable ibus-chewing and use it to type traditional Chinese.
- Activate the language bar, switch to English mode.
- When you deactivate the gedit text editor language bar(or panel) still will display Chinese language icon.
  In other words, language setting will still be setted as Chinese.
- When you want to type English, you just need switch from Chinese to English again.

**Comment by definite** [comment]
Fixed in ibus-chewing-1.4.14
